src/main/java/duke/Task.java

```diff
@@ -0,0 +1,36 @@
+package duke;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
```

Comment:
While this example is in the website, is there a better way to increase readability without using ternary expressions?